Saasu.API.Client/App.config

```diff
@@ -11,7 +11,7 @@
 
     <Saasu.API.Client.Config>
       <setting name="IgnoreSSLErrors" serializeAs="String">
-        <value>false</value>
+        <value>true</value>
```

Comment:
Why was this value changed?

Reply:
I'm getting connection SSL errors, though likely because I can't create an outbound connection.  The HTTPClient stops working after a few hours after launch.  This wasn't intended as part of the pull request.